### ISSUE TITLE
Next Playlist

### ISSUE THREAD

**Issue by fastender** [jishi/node-sonos-http-api#730]
How I can skip to the next playlist?

**Comment by jishi** [comment]
What do you mean? I'm not familiar with such a concept, Sonos has a Queue, when you change playlist you replace the queue...?

**Comment by jishi** [comment]
No, because there is no concept of what "next" playlist entails. Nor a list of playlists to begin with in this system. You need to explicitly specify what playlist it is supposed to play. You would have to roll your own next from a list or pick a random. 

Do note that it mostly support playing Sonos playlist, or playlists from music services that are added as favorites.
